packages/client-library-otel/src/instrumentation.ts

```diff
@@ -275,7 +276,13 @@ function setupTracerProvider(options: {
   const exporter = new OTLPTraceExporter({
     url: options.endpoint,
   });
-  provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
+  provider.addSpanProcessor(
+    new SimpleSpanProcessor(exporter),
```

Comment:
Outside of the scope of this pr, but when generating a large amount of traces, they get lost (aren't sent). Switching over to the BatchSpanProcessor fixes that partially, but now traces aren't always sent (immediately?)